composer.json

```diff
@@ -75,9 +72,26 @@
         },
         "config-plugin": {
             "tests": "config/tests.php",
-            "params": "config/params.php",
             "common": "config/common.php",
-            "console": "config/console.php"
+            "console": "config/console.php",
+            "tests-app": ["$common","$console","tests/App/config/main.php", "tests/App/config/console.php"]
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/yiisoft/mutex-file",
+            "no-api": true
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/yiisoft/mutex-db-mysql",
+            "no-api": true
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/yiisoft/mutex-db-pgsql",
+            "no-api": true
+        }
+    ]
```

Comment:
@samdark Could you publish those 3 packages to packagist?

Reply:
Done.